src/NoFrixion.MoneyMoov/Models/Invoices/Invoice.cs

```diff
@@ -0,0 +1,66 @@
+//  -----------------------------------------------------------------------------
+//   Filename: Invoice.cs
+// 
+//   Description: Invoice model
+// 
+//   Author(s):
+//   Donal O'Connor (donal@nofrixion.com)
+// 
+//   History:
+//   04 12 2023  Donal O'Connor   Created, Harcourt Street,
+//  Dublin, Ireland.
+// 
+//   License:
+//   Proprietary NoFrixion.
+//  -----------------------------------------------------------------------------
+
+namespace NoFrixion.MoneyMoov.Models.Invoices;
+
+public class Invoice
```

Comment:
I was thinking we'd use PayrunInvoice for this model (and it's actually already added under the Payruns directory). My thinking was there are different types of invoices that will be involved over time, Xero, bookkeeping etc. Using PayrunInvoice would make the intent more obvious.